Call init_transform_fs automatically in Brush framework

kvark opened the pull request:
(needs a try push)

bors-servo commented:
:umbrella: The latest upstream changes (presumably #2487) made this pull request unmergeable. Please resolve the merge conflicts.

glennw commented:
@kvark Should this be kept open / updated?

kvark commented:
IIRC, there was a Gecko reftest that I needed to investigate. I'll get back to it and revive this.

kvark commented:
Rebased and updated. Let's see how the try push goes: https://treeherder.mozilla.org/#/jobs?repo=try&revision=39f236e6bb48fd93d2c0f4fceca0ebe91c92bc95

glennw commented:
The try run looks good, I think - those two failues are from a previous box-shadow PR. There is one failing test on CI though. Otherwise, this looks good!

bors-servo commented:
:umbrella: The latest upstream changes (presumably #2534) made this pull request unmergeable. Please resolve the merge conflicts.

glennw commented:
Should we close this for now?

kvark commented:
I was just rebasing and re-testing it, let's not close yet.

bors-servo commented:
:umbrella: The latest upstream changes (presumably #2621) made this pull request unmergeable. Please resolve the merge conflicts.

gw3583 commented:
@kvark Should we keep this open?

kvark commented:
@gw3583 I think this is still on the table, yes.
